charter/opencdms_charter.md

```diff
@@ -29,11 +29,11 @@ For OpenCDMS to thrive, it needs to be seen to be agile and managed by consensus
 
 The OpenCDMS project will include modules providing specific data products to the WMO Information System, but assumes that WIS provides the interface to receive and transmit these products.
 
-The OpenCDMS project will include porting of end-user hydrology and climate applications which are currently operational in precursor systems (Climsoft, MCH, ClidDE, etc) but will not include development of new end-user applications. It is assumed that these will be provided by the (separate) WMO Climate Services Toolbox project, or others.
+The OpenCDMS project aims to offer hight quality hydrology and climate data, applications and services which help in adressing today national, regional and global needs. The OpenCDMS will ensure the porting of most functions in used by NMHSs, including the present open source CDMSs (Climsoft, MCH, ClidDE, etc.) and non-open source systems willing to participate to the project. It will also propose innovative methodologies and products that take into account available up-to-date standards and data. [This is because we need to attract also developed countries. They need to find an interest in the OpenCDMS project. Not to let them think that it is another system intended only for developing countries. It should be a system for all, open to all colaborative work on climate and hydrology]
```

Comment:
I like the first sentence, this should be put as a paragraph on its own.
I prefer the original text for the rest of this paragraph - it is aimed at limiting the scope to exclude applications which fit under the Climate Services Toolkit project so we are not duplicating effort elsewhere.